Automattic/woocommerce-payments
# feat: disable individual payment methods confirmation dialog

Fixes https://github.com/Automattic/woocommerce-payments/issues/1726

#### Changes proposed in this Pull Request









Added a confirmation dialog when the user has intent to delete a payment method.

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Click the "Delete" button next to a payment method in the list
- The modal appears
- Cancelling the modal does not remove the payment method
- Confirming the modal action removes the payment method

CC mobile view: https://d.pr/i/KzNWvT
CC desktop: https://d.pr/i/I03jgb
Giropay Desktop: https://d.pr/i/dyszPz

![2021-05-12 15 29 46](https://user-images.githubusercontent.com/273592/118040229-ec5c2600-b336-11eb-81e7-f1457461f713.gif)


-------------------
